### DISCUSSION TITLE
Display only threshold triggered rows

### DISCUSSION THREAD

**Discussion by p-malik** [grafana/grafana#38618] [Dashboards, Panels & Visualizations]
Is it possible to display only the threshold triggered rows for a repeating row dashboard? 
The threshold might have been triggered in any panel in the row.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
